lib/views/submit.js

```diff
@@ -509,7 +497,7 @@ function submitPost (req, res) {
             if (req.forceNoHTML || !req.accepts('text/html')) {
               res.status(200).type('text/plain').send('Successfully uploaded')
             } else {
-              res.redirect('/manage')
+              res.redirect('/user/' + encodeURIComponent(submissionData.createdBy.username) + '/' + collectionId + '/' + collectionId + '_collection/' + version)
```

Comment:
I can't remember -- can you submit into a public collection? If you can, would this break?

Reply:
Submission must go to a private collection.